python_modules/dagster/dagster/core/definitions/sensor.py

```diff
@@ -116,8 +116,8 @@ def repository_name(self) -> Optional[str]:
 SensorExecutionContext = SensorEvaluationContext
 
 
-class SensorDefinition:
-    """Define a sensor that initiates a set of runs based on some external state
+class ISensorDefinition:
+    """Base class for all types of SensorDefinitions.
```

Comment:
since this is more base class than interface, I would make it `BaseSensorDefinition` and probably use `ABC`